__tests__/app.test.js

```diff
@@ -236,6 +236,15 @@ describe('app', () => {
                 })
             });
         });
+        test('if an article has no comments responds with no comments msg and status 200', () => {
+            const idToSearch = 10
+            return request(app)
+            .get(`/api/articles/${idToSearch}/comments`)
+            .expect(200)
+            .then(({body}) => {
+                expect(body.msg).toEqual('No comments for this article');
```

Comment:
We should expect an empty array, not a string. Data consistency is very important, if everything goes correctly, we should only return one type of data